README.md

```diff
@@ -39,3 +39,13 @@ and apel-server.
 
 This is a messaging system, which has its own GitHub repository at
 [apel/ssm](https://github.com/apel/ssm)
+
+## Acknowledgements
+
+<span>
+  <img alt="STFC logo" src="https://github.com/GOCDB/gocdb/raw/dev/htdocs/web_portal/img/UKRI_STF_Council-Logo_Horiz-RGB_crop.png" height="57" />
+  <img alt="EU flag" src="https://github.com/GOCDB/gocdb/raw/dev/htdocs/web_portal/img/eu_flag_yellow_low_150.png" height="51" />
+  <img alt="EOSC-hub logo" src="https://github.com/GOCDB/gocdb/raw/dev/htdocs/web_portal/img/eosc-hub-v-web_150.png" height="57" />
```

Comment:
This assumes that this README will only ever be render to HTML for viewing in a web-browser, which is fine if that is true, but otherwise these should probably use Markdown images syntax.
```suggestion
![STFC logo](https://github.com/GOCDB/gocdb/raw/dev/htdocs/web_portal/img/UKRI_STF_Council-Logo_Horiz-RGB_crop.png)
![EU flag](https://github.com/GOCDB/gocdb/raw/dev/htdocs/web_portal/img/eu_flag_yellow_low_150.png)
![EOSC-hub logo](https://github.com/GOCDB/gocdb/raw/dev/htdocs/web_portal/img/eosc-hub-v-web_150.png)
```

Reply:
You can't control the image size in GitHub Markdown unfortunately. As I'm just being efficient and reusing what GOCDB has, then the actual fix would be to resize the images in the GOCDB repository. Could open an issue there, but otherwise I'm happy with how it looks here.